packages/projects-docs/pages/learn/_meta.json

```diff
@@ -1,11 +1,27 @@
 {
-  "introduction": "Introduction",
-  "getting-started": "Getting Started",
-  "sandboxes": "Sandboxes",
+  "index": "Introduction",
+  "getting-started": "Guides",
```

Comment:
Maybe we should update this slug to "guides" to remain consistent.

Reply:
:+1: